x/concentrated-liquidity/swaps.go

```diff
@@ -53,17 +53,23 @@ type SwapState struct {
 	// Global spread reward growth per-current swap.
 	// Initialized to zero.
 	// Updated after every swap step.
-	spreadRewardGrowthGlobal sdk.Dec
+	globalSpreadRewardGrowthPerUnitLiquidity sdk.Dec
+	// global spread reward growth
+	globalSpreadRewardGrowth sdk.Dec
```

Comment:
I agree with this rename, although we might want to carry it over to incentives in the future as well for consistency (`globalUptimeGrowth` currently represents `globalUptimeGrowthPerUnitLiquidity`)